process/templates/PROJECT_NAME_incubation_stage.md

```diff
@@ -14,7 +17,19 @@ The project should be able to show adoption by multiple parties and the adoption
 
 ### Governance
 Projects must have documented, initial project governance
-  * "link to governance documents"
+  * "link to governance documents/Charter"
+Projects must have met publicly at least 5 times
+  * "link to meeting agenda"
+Project must have a TAC sponsor
```

Comment:
I'm a bit confused. Per #262, I thought we had relaxed the TAC sponsor requirement for Projects/SIGs, while making it a requirement for WGs at the sandbox stage: https://github.com/ossf/tac/blob/main/process/TI-Gives%2BGets.md#givesrequirements